### PR TITLE
Fix xgboost example for hyperparameter tuning

### PR DESCRIPTION
**Pull request by richardsliu**
- Don't shuffle the input data to reduce randomness between trials
- Output mean_absolute_error in the format that katib metric collector expects

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/476)

### CODE REVIEW

File: xgboost_ames_housing/housing.py [hunk 1]
```diff
@@ -61,7 +62,7 @@ def train_model(train_X,
 def eval_model(model, test_X, test_y):
   """Evaluate the model performance."""
   predictions = model.predict(test_X)
-  print("MAE on test: {:.2f}".format(mean_absolute_error(predictions, test_y)))
+  print("mean_absolute_error={:.2f}".format(mean_absolute_error(predictions, test_y)))
```
Comment (jlewi) [inline xgboost_ames_housing/housing.py]: Should we use the logging module rather than print?

Comment (richardsliu) [reply xgboost_ames_housing/housing.py]: Fixed
